filebeat/module/nginx/_meta/config.full.yml

```diff
@@ -0,0 +1,33 @@
+- module: nginx
+  # Access logs
+  filesets.access:
+    enabled: true
+
+    # Format. Options are with_plugins and no_plugins
+    #var.pipeline: with_plugins
```

Comment:
I'm thinking if we should change this from `var` to `option`. I somehow dislike `var` as it is not very descriptive. It is a configuration option for the metricset. As these are "metricset" specific perhaps we can even remove the `var` completely and have it under `fileset.access.pipeline`. As most fileset have `pipeline` and `paths` these two could be basic options shared across all filesets like `ingest_pipeline`?